packages/contracts/src/contracts/current/protocol/Exchange/MixinExchangeCore.sol

```diff
@@ -159,7 +165,8 @@ contract MixinExchangeCore is
     function cancelOrder(
         address[5] orderAddresses,
         uint256[6] orderValues,
-        uint256 takerTokenCancelAmount)
+        uint256 takerTokenCancelAmount,
+        bytes signature)
```

Comment:
This signature does not include a nonce, so `cancelOrders` with this signature mechanism are prone to replays.

Reply:
In the current implementation we don't even need a signature here since `msg.sender` is required to be the maker.